cmd/strelaypoolsrv/gui/index.html

```diff
@@ -187,8 +187,8 @@ <h1>Relay Pool Data</h1>
       </div>
       <hr>
       <p>
-        This product includes GeoLite2 data created by MaxMind, available from
-        <a href="http://www.maxmind.com">http://www.maxmind.com</a>.
+        This product includes GeoLite2 data created by MaxMind, available at
+        <a href="http://www.maxmind.com">https://www.maxmind.com</a>.
```

Comment:
This was copied from their requirements

Reply:
Fixed. I have left the text alone, and just changed the URL to `https`.